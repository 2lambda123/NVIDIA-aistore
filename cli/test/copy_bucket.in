ais create bucket $BUCKET_1 $BUCKET_2
ais cp bucket $BUCKET_1/ $BUCKET_2/

ais cp bucket ais://$BUCKET_1 cloud://$BUCKET_2 // FAIL "Copying of cloud buckets not supported"

sleep 2
ais show xaction copybck $BUCKET_1/
ais show xaction copybck // FAIL "missing arguments "bucket name for xaction 'copybck'"
ais wait xaction copybck $BUCKET_2/
ais wait xaction copybck // FAIL "missing arguments "bucket name for xaction 'copybck'"
ais show xaction copybck $BUCKET_2/ // IGNORE
ais show xaction copybck $BUCKET_3/ // FAIL "Bucket with name "$BUCKET_3" does not exist."
