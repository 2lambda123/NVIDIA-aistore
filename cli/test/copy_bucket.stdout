$BUCKET_1 bucket created
$BUCKET_2 bucket created
Copying bucket $BUCKET_1 to $BUCKET_2 in progress.
To check the status, run: ais show xaction copybck $BUCKET_2
DaemonID	Kind	Bucket	Objects	Bytes	Start	End	Aborted
